devdoc/doublylinkedlist_requirements.md

```diff
@@ -20,13 +20,36 @@ typedef struct DLIST_ENTRY_TAG
     struct DLIST_ENTRY_TAG *Blink;
 } DLIST_ENTRY, *PDLIST_ENTRY;
 
+#define DLIST_MATCH_FUNCTION_RESULT_VALUES \
+    DLIST_MATCH_FUNCTION_MATCHING, \
+    DLIST_MATCH_FUNCTION_NOT_MATCHING, \
+    DLIST_MATCH_FUNCTION_ERROR
+
+MU_DEFINE_ENUM(DLIST_MATCH_FUNCTION_RESULT, DLIST_MATCH_FUNCTION_RESULT_VALUES);
+
+#define DLIST_CONDITION_FUNCTION_RESULT_VALUES \
+    DLIST_CONDITION_FUNCTION_SATISFIED, \
```

Comment:
>SATISFIED [](http://example.com/codeflow?start=29&length=9)

I'd probably had used TRUE/FALSE/ERROR. Sounds a bit more intuitive when writing the user code. But I guess this will work too, so I'll let you decide.

Reply:
changed to TRUE/FALSE